ci/test/00_setup_env_native_qt5.sh

```diff
@@ -15,5 +15,5 @@ export RUN_UNIT_TESTS_SEQUENTIAL="true"
 export RUN_UNIT_TESTS="false"
 export GOAL="install"
 export PREVIOUS_RELEASES_TO_DOWNLOAD="v0.15.0.0 v0.16.1.1 v0.17.0.3 v18.2.2 v19.3.0 v20.0.1"
-export BITCOIN_CONFIG="--enable-zmq --with-libs=no --enable-reduce-exports --disable-fuzz-binary LDFLAGS=-static-libstdc++ --with-boost-process"
+export BITCOIN_CONFIG="--enable-zmq --with-libs=no --enable-reduce-exports --disable-fuzz-binary --with-boost-process"
```

Comment:
why? Linter will fail then. See https://github.com/dashpay/dash/pull/5958#issuecomment-2034132559
It is required by `symbol-check` and `ci/test/00_setup_env_native_qt5.sh` uses gcc only (not clang).

Reply:
https://gitlab.com/dashpay/dash/-/jobs/6659301228 - failed as expected

Reply:
Didn't notice comment, figured it was skipped over. Thanks for pointing it out, commit dropped. Resolved in latest push.